website/pages/docs/autoscaling/policy.mdx

```diff
@@ -62,31 +62,92 @@ horizontal application scaling or horizontal cluster scaling.
   Detailed information on the configuration options can be found on the
   [Strategy Plugins][strategy_plugin_docs] page.
 
-### Example
+### Example in a Job
 
 A full example of a policy document that can be written into the Nomad task group
-`scaling` stanza or via a file within the `policy_dir` can be seen below.
+`scaling` stanza can be seen below.
 
 ```hcl
-min     = 2
-max     = 10
-enabled = true
+job "example" {
+  group "app" {
+    scaling {
+      min     = 2
+      max     = 10
+      enabled = true
+
+      policy {
+        evaluation_interval = "5s"
+        cooldown            = "1m"
+
+        check "active_connections" {
+          source = "prometheus"
+          query  = "scalar(open_connections_example_cache)"
+
+          strategy "target_value" {
+            target = 10
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Example in a File
+
+An example of a policy document that can be placed in a file within the
+`policy_dir` can be seen below. Multiple policies can be defined in the same
+file using multiple `stanza` blocks.
```

Comment:
Is this supposed to be `scaling` rather than `stanza`?

Reply:
@jrasell because @lgfa29 is out I've resolved this in c3ae932